contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -547,7 +536,7 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
     bool staleFallback = stalenessSeconds > 0;
     uint256 timestamp;
     int256 weiPerUnitLink;
-    (, weiPerUnitLink, , timestamp, ) = LINK_ETH_FEED.latestRoundData();
+    (, weiPerUnitLink, , timestamp, ) = SubscriptionAPI.LINK_ETH_FEED.latestRoundData();
```

Comment:
*nit I think you can just do below since SubscriptionAPI is inherited:
```suggestion
    (, weiPerUnitLink, , timestamp, ) = LINK_ETH_FEED.latestRoundData();
```

Reply:
Sure, trying this and compiling

Reply:
Made these changes, compiled again and ran all tests. All tests pass.